### DISCUSSION TITLE
Rookie question(s)<title>

### DISCUSSION THREAD

**Discussion by MRM256** [community/community#65768] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have a project I created using VB.NET (Visual Studio 2017) that I wish to share as an open-source project, so others can add their own functionality, such as a Log-in tab page for a different database system.

Questions:
1 - Which license do I use for a project created using VB.NET?
2 - Can someone teach me how to correct my Classes? 
3 - Add the ability to use a pull-down menu for things like Selecting a state name for an address?

These are just some a few that come to mind.

Any Help is appreciated,
MRM256

**Comment by jesse-sealand** [comment]
No, it's an excellent question. You can absolutely use the **MIT License** and to understand why there are two things you need to know. 

1. ".NET is open-source and under the [.NET Foundation](https://www.dotnetfoundation.org/). The .NET Foundation is an independent organization to foster open development and collaboration around the .NET ecosystem." [source](https://dotnet.microsoft.com/en-us/platform/free)

2. "The MIT License (MIT) Copyright (c) .NET Foundation and Contributors" [source](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT)
